### ISSUE TITLE
Removing particles

### ISSUE THREAD

**Issue by surgura** [OceanParcels/parcels#1591]
Let me know if this feature exists; I think it does not?
It is possible to release particles later than the starting time of the fieldset, by using the 'time' parameters in a ParticleSet. If possible, I would like to set an end time for the particles as well.

**Comment by erikvansebille** [comment]
Yes, you can remove particles using `particle.delete()` in a Kernel. 

And yes, you can start a particle later by setting its `time` attribute when you create the ParticleSet, in a similar way that you would set the lon, lat or depth. See the [delayed start tutorial](https://docs.oceanparcels.org/en/latest/examples/tutorial_delaystart.html).

And yes, you can remove a particle after a certain time by adding e.g. an `age` Variable, and adding a Kernel like (not tested)
```python
def Aging(particle, fieldset, time):
    particle.age += particle.dt
    if particle.age > max_age:
        particle.delete()
````
